ports/renesas-ra/machine_uart.c

```diff
@@ -557,6 +567,20 @@ STATIC mp_uint_t machine_uart_ioctl(mp_obj_t self_in, mp_uint_t request, uintptr
         if ((flags & MP_STREAM_POLL_WR) && uart_tx_avail(self)) {
             ret |= MP_STREAM_POLL_WR;
         }
+    } else if (request == MP_STREAM_FLUSH) {
+        // The timeout is estimated using the buffer size and the baudrate.
+        // Take the worst case assumptions at 13 bit symbol size times 2.
+        // t.b.d. get actial TX buffer size
```

Comment:
Can this comment line be removed, since the actual TX buffer size is being retrieved via `uart_tx_txbuf`?

Reply:
(if the line stays, typo fix actial -> actual)

Reply:
Sure. This can go. I'll remove the last line of the comment.

Reply:
... and updated.